redis/mutation_utils.cc

```diff
@@ -68,6 +68,11 @@ future<> write_strings(service::storage_proxy& proxy, redis::redis_options& opti
     return proxy.mutate(std::vector<mutation> {std::move(m)}, write_consistency_level, timeout, nullptr, permit);
 }
 
+future<> write_strings_pairs(service::storage_proxy& proxy, redis::redis_options& options, std::map<bytes, bytes>&& data_map, long ttl, service_permit permit) {
```

Comment:
Why do you call this "strings_pairs"? Isn't this like write_strings, just a list of them instead of just one?
By the way, why do you use a std::map<> instead of just an std::vector or std::list - you don't really need these the keys sorted or (I think) duplicates avoided - can't you just take the same list of items you got from the user, and write them in the same order?

Reply:
The pair is the key and value, after parsing the arguments, I organizes them to multiple pairs of key - value. It's fine to pass two vectors or lists by two arguments as past.

Reply:
Ok, let's ignore these issues now, but for future consideration:
1. write_strings also takes a "pair" in this sense, so it's strange to call just one of the function "pairs"
2. You don't really need an std::map here, a std::list<std::pair>, or similar, would have been enough.

Reply:
I will fix this by another pr.